.travis.yml

```diff
@@ -8,6 +8,9 @@ env:
   global:
     - PYENV_PY36_VERSION=3.6.8
     - PYENV_PY37_VERSION=3.7.2
+    - PYENV_ROOT="${PYENV_ROOT:-${HOME}/.pants_pyenv}"
```

Comment:
Why would you want to accept a pre-existing value here?

Reply:
Good question. Trusty and Xenial predefine `PYENV_ROOT` and overriding it means their Python 3 will no longer work. Added a comment to explain this, and will also update Pants repo to explain this as well.